ERCS/erc-xxxx.md

```diff
@@ -0,0 +1,244 @@
+| paymasterValidationGasUsed | QUANTITY        | The amount of gas actually used by the paymaster validation frame              |
+| executionGasUsed           | QUANTITY        | The amount of gas actually used by the execution frame                         |
+| postOpStatus               | QUANTITY        | 1 (success), 0 (failure), or `null` (did not run) status of the `postOp` frame |
+| postOpGasUsed              | QUANTITY        | The amount of gas actually used by the paymaster `postOp` frame                |
+| validationLogs             | ARRAY           | Array of log objects, which this transaction'S VALIDATION FRAME generated.     |
+| validationLogsBloom        | DATA, 256 Bytes | Array of log objects, which this transaction'S VALIDATION FRAME generated.     |
+
+Continued, these fields are shared by all transaction types:
+
+| Name              | Type            | Value                                                                                      |
+|-------------------|-----------------|--------------------------------------------------------------------------------------------|
+| transactionHash   | DATA, 32 Bytes  | Hash of the transaction.                                                                   |
+| transactionIndex  | QUANTITY        | Integer of the transactions index position in the block.                                   |
+| blockHash         | DATA, 32 Bytes  | Hash of the block where this transaction was in.                                           |
+| blockNumber       | QUANTITY        | Block number where this transaction was in.                                                |
+| cumulativeGasUsed | QUANTITY        | The total amount of gas used when this transaction was executed in the block.              |
+| effectiveGasPrice | QUANTITY        | The sum of the base fee and tip paid per unit of gas.                                      |
+| gasUsed           | QUANTITY        | The amount of gas used by this specific transaction alone.                                 |
+| contractAddress   | DATA, 20 Bytes  | The contract address created, if the transaction was a contract creation, otherwise `null` |
+| logs              | ARRAY           | Array of log objects, which this transaction'S EXECUTION FRAME generated.                  |
+| logsBloom         | DATA, 256 Bytes | Bloom filter for light clients to quickly retrieve related logs.                           |
+| type              | QUANTITY        | Integer of the transaction type                                                            |
+| status            | QUANTITY        | Either 1 (success) or 0 (failure) status of the execution frame                            |
+
+#### Create a new JSON-RPC API method `eth_executeRip7560Transaction`
+
+Executes the entire RIP-7560 transaction in memory without broadcasting it or including it in a block.
```

Comment:
note that unlike eth_call , rip7560 transaction call can depend on its validation, so it can't be called in isolation from its deployment/validation

Reply:
Added.